stack.yaml

```diff
@@ -10,6 +10,10 @@ packages:
 - hie-ghc-tree
 - hie-hare
 - hie-docs-generator
+- location:
```

Comment:
Its for the "no-cabal" project case, needing https://github.com/alanz/HaRe/commit/48102f8899a4cd837fa0891620d5264bbf09d7fe